Банки

DamnedDeus opened the pull request:
Шкут Р.В., гр.821703

Из всех отделений доступно для формализации 5-10 % в лучшем случае, остальное имеет одинаковый тэг "name" и отдельно не ищется. Нужна привязка к улицам/зданиям, хотя не уверен что это поможет для всех прецедентов.

Не знаю куда кидать списки банков, subject_domain_belarus - не слишком подходит.

Во 08.02 и 0.9.03 физически не было доступа к ПК по уважительной причине и, соответственно, проекту, так что удалось доделать только сейчас.

DamnedDeus committed:
description

Добавлено описание и список понятий

DamnedDeus committed:
all concepts

Выгрузка всех понятий

Vikort commented on kb/section_subjdomain_openstreetmap/banking institution/banks_info_brest.scs:
```diff
@@ -0,0 +1,159 @@
+=>nrel_city:
+    pinsk;;
+
+//belveb_brest_cbu_102 <- concept_bank;
+//<- concept_map_object;
+//=>nrel_main_idtf:
+//    [БелВЭБ РКЦ "Припять"]
+//    (* <- lang_ru;; <- name_ru;; <- name;;*);
+//=>nrel_operator:
+//	belveb_brand;
+//=>nrel_country:
+//    belarus;
+//=>nrel_region:
+//    brest_region;
+//=>nrel_district:
+//    pinsk_district;
+//=>nrel_city:
+//    pinsk;;
```
раскомментируй
находит, только если вставить `\` перед ковычками, а так понятие нормально описано

Vikort reviewed (commented):
Раскомментируй, то что не находит, или удали
Первая часть принята
Пр смержим после принятия всех частей

DamnedDeus committed:
ui_menu add agents

DamnedDeus committed:
Add agents

Добавлены агенты:
агент поиска по городу
агент поиска по типу 
агент поиска по брэнду

DamnedDeus committed:
deleting comments

DamnedDeus committed:
Delete problem-solver/py/services/ExampleModule directory
